contributing/dependencies.md

```diff
@@ -0,0 +1,12 @@
+# Dependencies
```

Comment:
I'm not sure if this file is necessary. It was something was added by Gunnar a while back. Not sure if this is actually used. I am also not sure if this is up to date.

Reply:
Deleted.